software/analytics/ibm-wml-ce.rst

```diff
@@ -163,12 +174,12 @@ Performance Profiling
 
 There are several tools that can be used to profile the performance of a
 deep learning job. Below are links to several tools that are available
-as part of the ibm-wml-ce module.
+as part of the ibm-wml-ce and open-ce module.
 
 NVIDIA Profiling Tools
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The ibm-wml-ce module contains the nvprof profiling tool. It can be used to
+The ibm-wml-ce and open-ce module contains the nvprof profiling tool. It can be used to
```

Comment:
"module contains" -> "modules contain"

Reply:
Thanks @verolero86, I have fixed all typos.

Reply:
Thank you @jqyin!